modules/mobile/spec/workers/pre_cache_appointments_job_spec.rb

```diff
@@ -18,8 +18,6 @@
 
   after(:all) { VCR.configure { |c| c.cassette_library_dir = @original_cassette_dir } }
 
-  let(:user) { FactoryBot.build(:iam_user) }
```

Comment:
In all of these tests, I've switched to using `User` instead of `IAMUser` as the base case because 1) it's simpler to set up 2) it's the way of the future. So I've done most of the testing with User and also confirmed that it works with IAMUser.